### DISCUSSION TITLE
Community group?

### DISCUSSION THREAD

**Discussion by Yoddikko** [arichornlover/uYouEnhanced#670] [Q&A] [answered]
First of all I'd like to thank you @arichornlover for the hard work on this repo. I'd like to know if it exists a group on telegram or discordto engage with the community and the developer/s. Thanks

**Comment by yodaluca23** [comment]
There is an unofficial telegram group I believe look around here for the link someone’s posted it.

**Reply by Yoddikko** [reply]
Thanks. I'd like to find a group where they post the latest IPAs of uYouEnhanced.. since I tried to build it myself but can't make it work

**Reply by arichornlover** [reply]
I’m pretty sure the unofficial uYouEnhanced Telegram has IPA files of the latest release there.

**Reply by samuelawachie** [reply]
> Thanks. I'd like to find a group where they post the latest IPAs of uYouEnhanced.. since I tried to build it myself but can't make it work

Just use driftywinds repo. Search for it

**Reply by Yoddikko** [reply, marked as answer]
https://t.me/c/2107009220/17

**Reply by LCY000** [reply]
Is there anyone who can provide ipa files?
It seems difficult to make ipa using windows system

This link doesn't seem to work
Thanks!
